### PR TITLE
assisted installer: add assisted-grafana to CI

### PR DESCRIPTION
**Pull request by rccrdpccl**
Move grafana dashboard for assisted-installer to separate repository. This way we can simplify slightly CI execution for both projects and clearer history, etc.

/cc @gamli75 @osherdp

### CODE REVIEW

File: core-services/prow/02_config/openshift-assisted/assisted-grafana/_prowconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+tide:
+  queries:
+  - labels:
+    - approved
+    - lgtm
+    missingLabels:
+    - backports/unvalidated-commits
+    - bugzilla/invalid-bug
```
Comment (osherdp) [inline core-services/prow/02_config/openshift-assisted/assisted-grafana/_prowconfig.yaml:8]: ``bugzilla/invalid-bug`` seems redundant here, no?
similar question to the ``jira/invalid-bug`` label, as we don't have the jira plugin enabled for this repo

File: ci-operator/config/openshift-assisted/assisted-grafana/openshift-assisted-assisted-grafana-master.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
Comment (osherdp) [inline ci-operator/config/openshift-assisted/assisted-grafana/openshift-assisted-assisted-grafana-master.yaml:5]: can we upgrade to 1.18?
just so we won't have a too old image

**Review by osherdp — CHANGES_REQUESTED** [review]
just two small things

File: core-services/prow/02_config/openshift-assisted/assisted-grafana/_prowconfig.yaml (hunk 1 above)
Comment (rccrdpccl) [reply core-services/prow/02_config/openshift-assisted/assisted-grafana/_prowconfig.yaml:8]: Sure, this was copy-pasted from other repos so I will also change that in another PR

**Comment by rccrdpccl** [comment]
rehearse will likely fail as the test is not in master https://github.com/openshift-assisted/assisted-grafana/tree/add-readme

**Review by osherdp — APPROVED** [review]

**Comment by rccrdpccl** [comment]
@osherdp are you aware if this failure is due to my config?

**Comment by osherdp** [comment]
> @osherdp are you aware if this failure is due to my config?

Seems like it's because we still enable bugzilla plugin globally:
https://github.com/openshift/release/blob/dec7a530455aac1f30bdadca5045d685ab8a93a2/core-services/prow/02_config/openshift-assisted/_pluginconfig.yaml#L62

**Review by osherdp — APPROVED** [review]

**Comment by rccrdpccl** [comment]
> Seems like it's because we still enable bugzilla plugin globally:

Thanks, I feel dumb for not seeing it :sweat: 
I take it we can remove it globally as, AFAIU, we do not use it in any longer (and even more so in openshift-assisted)

**Comment by rccrdpccl** [comment]
> Sure, this was copy-pasted from other repos so I will also change that in another PR

I guess I need to change them in this PR after all :sweat_smile:

File: core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml [hunk 3]
```diff
@@ -26,7 +26,6 @@ tide:
     - lgtm
     missingLabels:
     - backports/unvalidated-commits
-    - bugzilla/invalid-bug
     - do-not-merge/hold
     - do-not-merge/invalid-owners-file
     - do-not-merge/work-in-progress
```
Comment (osherdp) [inline core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml:31]: should we remove ``jira/invalid-bug`` as well?

**Review by osherdp — CHANGES_REQUESTED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**